src/ProtocolV3TestBase.sol

```diff
@@ -2,36 +2,30 @@
 pragma solidity >=0.7.5 <0.9.0;
 
 import 'forge-std/Test.sol';
-import {
-  IAaveOracle,
-  IPool,
-  IPoolAddressesProvider,
-  IPoolDataProvider,
-  IDefaultInterestRateStrategy,
-  DataTypes,
-  IPoolConfigurator
-} from 'aave-address-book/AaveV3.sol';
-
-import { ReserveConfiguration } from 'aave-v3-core/contracts/protocol/libraries/configuration/ReserveConfiguration.sol';
-import { WadRayMath }           from 'aave-v3-core/contracts/protocol/libraries/math/WadRayMath.sol';
-import { IAToken }              from 'aave-v3-core/contracts/interfaces/IAToken.sol';
-import { IStableDebtToken }     from 'aave-v3-core/contracts/interfaces/IStableDebtToken.sol';
-import { IVariableDebtToken }   from 'aave-v3-core/contracts/interfaces/IVariableDebtToken.sol';
-
-import { IERC20 }    from './interfaces/IERC20.sol';
-import { SafeERC20 } from './libraries/SafeERC20.sol';
-
-import { ICapAutomator }  from './interfaces/ICapAutomator.sol';
-import { ProxyHelpers }   from './libraries/ProxyHelpers.sol';
-import { CommonTestBase } from './CommonTestBase.sol';
 
-interface IERC20Detailed is IERC20 {
-  function name() external view returns (string memory);
+import { ReserveConfiguration }         from 'sparklend-v1-core/contracts/protocol/libraries/configuration/ReserveConfiguration.sol';
+import { WadRayMath }                   from 'sparklend-v1-core/contracts/protocol/libraries/math/WadRayMath.sol';
+import { DataTypes }                    from 'sparklend-v1-core/contracts/protocol/libraries/types/DataTypes.sol';
```

Comment:
Lets separate the `libraries` into their own group and put it below `interfaces`

Reply:
done